### DISCUSSION TITLE
FriendlyCaptcha validation with disabled Javascript

### DISCUSSION THREAD

**Discussion by muratbinerbay** [dachcom-digital/pimcore-formbuilder#487] [General]
Hi all,

we realised something about the FriendlyCaptcha validation and wanted to post it, incase it helps anyone else:
- Currently we have the `FriendlyCaptcha` field in a `Fieldset` with the form `Submit` field (so only 2 fields in this fieldset).
- If you disable Javascript on the page then you can submit the form, even though FriendlyCaptcha never got to solve the puzzle.

After debugging we saw that the `FriendlyCaptchaValidator` class was never called, I assume because there is no input field generated in the fieldset. 

We were able to solve the problem by adding a `Hidden Type` to the fieldset that contained the `FriendlyCaptcha`. So if Javascript is disabled on the page, the `FriendlyCaptchaValidator` will catch it.

**Comment by solverat** [comment]
@muratbinerbay Thanks for the share! I guess we can live with that, since all those providers require javascript to populate the score field.
